### ISSUE TITLE
If there is Grass on Grass Blocks they wont turn into meat blocks

### ISSUE THREAD

**Issue by AndreDoruk** [AurySystem/Vegant-Meat-Fields#2]
![2023-04-23_15 42 01](https://user-images.githubusercontent.com/81973147/233853753-db8227e1-30a1-4a7b-9f47-25999a234718.png)
It goes for flowers too but Grass (and Tall Grass) being more prominent its more noticeable

**Comment by AurySystem** [comment]
yeah, that's the same for how moss growing works, we'll refactor the feature eventually

**Comment by AndreDoruk** [comment]
Thank you that would be amazing as it makes it look very ugly
